### DISCUSSION TITLE
Supported Devices seem nonexistent

### DISCUSSION THREAD

**Discussion by danbraun** [Koenkk/zigbee2mqtt#13352] [Devices]
After spending about a half an hour googling various models from supported devices, everything seems to be discontinued or nonexistent. Is there a way or has anyone any suggestions as to how I can narrow the search  or find anything online from supported devices? Smart bulbs with my main target.

**Comment by sjorge** [comment]
I'd start the otherway around check  what is available on your local amazon or equalivant or hardware store, then check if it is supported.

Most (all AFAIK) IKEA Trådfri bulbs available locally for me are supported, as is most Philips Hue stuff.
I've also ordered a mix of Innr and Osram from amazon which were also on the supported list.
